db/rocksdb_ethereumtype.go

```diff
@@ -53,19 +53,58 @@ func packAddrContracts(acs *AddrContracts) []byte {
 			l = packBigint(&ac.Value, varBuf)
 			buf = append(buf, varBuf[:l]...)
 		} else if ac.Type == bchain.NonFungibleToken {
-			l = packVaruint(uint(len(ac.Ids)), varBuf)
+			i := 0
+			ids := []*big.Int{}
+			ac.Ids.Range(func(key, val any) bool {
```

Comment:
Could probably reduce to single loop by skipping the length byte, then adding back to whatever index it would have been at since we have to increment `i` during `Range` for length. Figured, keep it simple to avoid possibly messing something up..